lib/components/icons/Icon/Icon.migration.md

```diff
@@ -0,0 +1,13 @@
+# Icon Migration Guide
+
+## API Changes
+
+- If you need a custom icon size, use `size="fill"` to make the icon fill its container.
```

Comment:
Icons previously supported the [full typographic hierarchy](https://github.com/seek-oss/seek-style-guide/blob/master/react/private/withTextProps.js#L10), currently only support the text sizes.